### PR TITLE
Fix cuda build error where it fails to find the `cuda_runtime_api.h` file

### PR DESCRIPTION
**Pull request by jmackay2**

### CODE REVIEW

File: cmake/pcl_find_cuda.cmake [hunk 1]
```diff
@@ -56,8 +54,8 @@ if(CUDA_FOUND)
     set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -gencode arch=compute_${ver},code=compute_${ver}")
     message(STATUS "CMAKE_CUDA_FLAGS: ${CMAKE_CUDA_FLAGS}")
     
-    add_library(pcl_cuda INTERFACE)
-    target_include_directories(pcl_cuda INTERFACE ${CUDA_TOOLKIT_INCLUDE})
-    
   endif ()
+
+  add_library(pcl_cuda INTERFACE)
```
Comment (kunaltyagi) [inline cmake/pcl_find_cuda.cmake:59]: @larshg PTAL (since the previous comment regarding empty project was added by you)

**Comment by larshg** [comment]
Which target failed to build?

**Comment by larshg** [comment]
I cannot reproduce any errors in with the mentioned setup.

**Comment by jmackay2** [comment]
> Which target failed to build?

kinfu

**Pull request closed without merging**